scripts/fuzz_passes.py

```diff
@@ -105,7 +105,7 @@ def simplify(passes):
         try:
           apply_passes(smaller)
           assert run() == normal
-        except:
+        except Exception, e:
```

Comment:
Modern style is `except Exception:` or `except Exception as e:`

Reply:
Thanks, updated.